.gitignore

```diff
@@ -57,3 +57,5 @@ buck-out/
 
 # CocoaPods
 /ios/Pods/
+
+.env
```

Comment:
You only needed to add an example file, to identify how the key should be entered.